var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -22,11 +22,12 @@ class Singularity(SingularityBase):
 
     homepage = "https://singularity.hpcng.org/"
     url      = "https://github.com/hpcng/singularity/releases/download/v3.6.4/singularity-3.6.4.tar.gz"
-    git      = "https://github.com/hpcng/singularity.git"
+    url      = "https://github.com/hpcng/singularity/releases/download/v3.8.3/singularity-3.8.3.tar.gz"
```

Comment:
```suggestion
    url = "https://github.com/hpcng/singularity/releases/download/v3.8.3/singularity-3.8.3.tar.gz"
    git      = "https://github.com/hpcng/singularity.git"
```

Reply:
Weird I couldn't apply this change - but just pushed from local branch!

Reply:
Uh yeah, i also got the impression the suggestion didnt make sense, maybe a bug in github actually